### PR TITLE
in my settings, i should be able to disguise myself from the leaderboard/having a profile

### PR DESCRIPTION
**Pull request by owocki**
##### Description
in my settings, i should be able to disguise myself from the leaderboard/having a profile

also sends an email when a user puts feedback in the comments

In the future, we should maybe break apart the email settings / gitcoin profile settings (as there are two models: EmailSubscriber / Profile models)

##### Checklist

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Affected core subsystem(s)
email settings, profile

##### Testing
tested it

##### Refers/Fixes
https://github.com/gitcoinco/web/issues/710

### CODE REVIEW

File: app/external_bounties/management/commands/sync_bounty0x.py [hunk 1]
```diff
@@ -43,6 +43,7 @@ def handle(self, *args, **options):
 
         response = requests.get(url, headers=headers)
         bounties = response.json()
+        print(len(bounties))
```
Comment (thelostone-mc) [inline app/external_bounties/management/commands/sync_bounty0x.py]: @owocki  take this away ?

File: app/marketing/management/commands/assemble_leaderboards.py [hunk 2]
```diff
@@ -33,6 +33,7 @@
 quarterly_cutoff = timezone.now() - timezone.timedelta(days=90)
 yearly_cutoff = timezone.now() - timezone.timedelta(days=365)
 
+
```
Comment (thelostone-mc) [inline app/marketing/management/commands/assemble_leaderboards.py]: @owocki  extra line added ?

**Comment by owocki** [comment]
> In the future, we should maybe break apart the email settings / gitcoin profile settings (as there are two models: EmailSubscriber / Profile models)

Any thoughts on this?  I'm gonna take another pass here this afternoon and it'd be good to have this fleshed out.

File: app/marketing/management/commands/assemble_leaderboards.py (hunk 2 above)
Comment (owocki) [reply app/marketing/management/commands/assemble_leaderboards.py]: i think 2 line breaks is the standard before top level methods in pep8

File: app/external_bounties/management/commands/sync_bounty0x.py (hunk 1 above)
Comment (owocki) [reply app/external_bounties/management/commands/sync_bounty0x.py]: da73e3cd79cbdc8e7114e8bdde8577571b455a68

**Comment by owocki** [comment]
> In the future, we should maybe break apart the email settings / gitcoin profile settings (as there are two models: EmailSubscriber / Profile models)


follow up ticket for the above

https://github.com/gitcoinco/web/issues/795

**Comment by owocki** [comment]
i went in to resolve some of the merge conflicts and it was borderline impossible to work inside the view as it was.

769033eb1b020a9a98f11b65fff7011da726ea06 <=== ended up just implementing https://github.com/gitcoinco/web/issues/795 so that we could work with the information in a sane way in the future

**Pull request merged**